english/stargate/GateDial.md

```diff
@@ -0,0 +1,5 @@
+Before you stands the Stargate's control panel, giving you access to the universe. One last symbol remains to activate the dial, which symbol will you choose?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.